docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,530 @@
+Still, even in the target device, nothing more than BasicUI will happen as expected.
+
+
+<a name="ui"></a>
+## Designing a Simple UI
+
+The reason that there is no change even after installing the addon above, is that a callback function registered to addon event has only code to print out a log to console.
+
+Now, let's make some changes which is viewable in a web application.
+
+We hope you are familiar with the [basic knowledge in addons](../guides/applications/addon.md).
+
+### Showing a Pop-up on Resume of the Web Application
+
+In this example, we will catch the timing of resuming event in the web application. On the timinig, we will show up a pop-up with message 'Hello. Resume occurred!!'
+
+First, let's look into the JavaScript file `tmplDefault.js` at `shared/res/`.
+
+![JavaScript file for template](media/addon_template_js_file.png)
+
+Based on the knowledge you got from the introduction page for addon linked above, you seems to know the below for the addon :
+
+| Used Event     | Matched Callback    |
+| -------------- | ------------------- |
+| lcPrelaunch | prelaunch |
+| lcResume | resume |
+| lcSuspend | suspend |
+| lcQuit | quit |
+
+
+For each callback, it has only `console.log()` as implementation.
+
+Now, we will focus on only `lcResume` to show up a pop-up with some message. You can remove all the lines related to other events except `lcResume`.
+
+![JavaScript file for resume](media/js_for_resume.png)
+
+You need to use `addonapi` which includes APIs for addons. To show the pop-up, you can use `addonapi.window.showMessageBox()`. Here's a usage of it.
+
+```
+const addonAPI = require('addonapi');
+
+resume(winId) {
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+    
+
+By adding the lines above, the JavaScript file will look like this :
+
+![Code for resume pop-up](media/hello_resume_code.png)
+
+With the changed file, you need to build a signed package again.
+
+Before installing the newly built package, you have to remove the installed addon first.
+
+In Addon Settings App, you can see the 'X' button in red box in the row of the installed addon list. To remove the installed addon, you need to press the button.
+
+Now, the changed addon package with pop-up on resume event will be installed if you follow the steps you did with the template package before.
+
+When you run the Basic UI web application, you will see the pop-up as below.
+
+**Figure: Addon Pop-up on Resume Event in the Basic UI**
+
+![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+### Loading an HTML File from Addon
+
+In an addon, you can load your own html file if you need. We are going to write an HTML file which shows `Hello, I'm an Addon`.
+
+All files related to an addon should be located under `shared\res` of your addon project. Therefore, the added HTML file should be there, too.
```

Comment:
**Note**: From here, the content is written more in a blog language. Request the author to please maintain the writing style as per the Tizen docs. Also, since this file is too huge, could we have the Example part in a separate page?
@mijin85cho, please provide your opinion as well.

Reply:
done for rephrasing.

For changing the blog language to Tizen docs' language,
I'll try but I don't understand it exactly. It will be helpful if you attach any guide page for that.

For separating the contents, I'll wait for the comment from @mijin85cho